radiation_obs/README.md

```diff
@@ -0,0 +1,7 @@
+# Radiative fluxes data from CERES
+
+This artifact contains monthly mean top-of-the-atmosphere radiative fluxes from the 
+Clouds and Earth's Radiant Energy Systems (CERES) Energy Balanced and Filled (EBAF) from March 2000 to March 2023. 
+The data can be downloaded from the data repository [Will and Schneider 2024](https://data.caltech.edu/records/z24s9-nqc90).
+
+License: Creative Commons Attribution 4.0 International
```

Comment:
Can you please describe the dataset more?

What observables, units, dates, conventions, et cetera?

Reply:
The dates are already in the README. I added variables and conventions, as well as the units (the units are also described in the netcdf file).